### PR TITLE
tf instructions for rosa classic

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

File: content/rosa/terraform/classic.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    ```bash
+    terraform init
+
+    terraform plan -out tf.plan
+
+    terraform apply tf.plan
+
+    ```
+
+    If everything goes to plan, after about 20 minutes you should have a cluster available to use.
```
Comment (daxelrod-rh) [inline content/rosa/terraform/classic.md:48]: For a Classic cluster, this is probably more like 45 minutes

File: content/rosa/terraform/hcp.md [hunk 2]
```diff
@@ -37,12 +37,9 @@ This guide will walk you through deploying a ROSA cluster using Terraform.  This
...
     ```bash
-    terraform init
-
-    terraform plan -out tf.plan
-
-    terraform apply tf.plan
-
+    terraform init && \
+      terraform plan -out tf.plan && \
+      terraform apply tf.plan
```
Comment (daxelrod-rh) [inline content/rosa/terraform/hcp.md:42]: Not a blocker, just curious: Why this change in this file but not the other one?

File: content/rosa/terraform/classic.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    > Note: You may want to customize some of these settings to match your needs. see the `variables.tf` file for options.
+
+    ```bash
+    export TF_VAR_token="$(jq -r .refresh_token ~/.config/ocm/ocm.json)"
+    export TF_VAR_cluster_name="$(whoami)"
+    export TF_VAR_admin_password='Passw0rd12345!'
+    export TF_VAR_developer_password=''
+    export TF_VAR_private=false
+    export TF_VAR_ocp_version=4.15.11
+    export TF_VAR_hosted_control_plane=false
```
Comment (daxelrod-rh) [inline content/rosa/terraform/classic.md:33]: Why have two nearly identical guides for HCP and Classic that mostly differ based upon whether this parameter is true or false?

**Review by daxelrod-rh — DISMISSED** [review]
This generally looks good, the only blocker I see is the time estimate for spinning up a Classic cluster.

I wonder, though, what's the purpose of two almost identical guides for HCP or Classic, rather than just one?

File: content/rosa/terraform/classic.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+tags: ["ROSA", "terraform"]
+authors:
+  - Paul Czarkowski
+---
+
+This guide will walk you through deploying a ROSA cluster using Terraform.  This is a great way to get started with ROSA and to automate the deployment of your clusters.
+
+{{< readfile file="/content/rosa/terraform/tf-prereqs.md" markdown="true" >}}
+
+## HCP ROSA Cluster
```
Comment (scottd018) [inline content/rosa/terraform/classic.md:13]: Should be changed to 'Classic'

**Review by scottd018 — APPROVED** [review]

File: content/rosa/terraform/classic.md (hunk 1 above)
Comment (scottd018) [reply content/rosa/terraform/classic.md]: Committed and fixed the time values as well as the header to change from `HCP` > `ROSA Classic`
